source/models/table.py

```diff
@@ -47,22 +53,65 @@ def set(self, row):
             old_row.delete()
         row_obj.serialize()
 
-    
-    def get(self):
-        pass
-    
+
+    def get_similar_primarykeys(self, query):
+        primarykeys = []
+        for index_name in query:
+            if index_name in self.table_metadata.index_keys:
+                primarykeys = Index(self, index_name = index_name, index_value = query[index_name]).get_primary_key(index_value = query[index_name])
+                break
+        return primarykeys
+ 
+
+    def get(self, query):
+        self.__colomns_name_validate(query)
```

Comment:
I asked @josephShokry  to move this in Row as a static method

Reply:
I think it will be better to put it on class table as when I want to get or delete a query I check it in first before I go to raw class

Reply:
as query shouldn't have column doesn't in table metadata